soc/nordic/nrf53/Kconfig

```diff
@@ -163,10 +163,11 @@ config NRF_SPU_RAM_REGION_SIZE
 	  RAM region size for the NRF_SPU peripheral
 
 config SOC_NRF_GPIO_FORWARDER_FOR_NRF5340
-	bool
+	bool "Forward GPIO pins to network core"
```

Comment:
Why make this option user configurable? Users can disable this feature by disabling "nordic,nrf-gpio-forwarder" nodes in devicetree and I think it's more consistent to allow doing it only this way.